sw/device/lib/testing/uart_testutils.h

```diff
@@ -34,8 +34,9 @@ typedef struct uart_cfg_params {
   uint32_t base_addr;
   uint32_t peripheral_id;
   uint32_t irq_tx_watermark_id;
+  uint32_t irq_tx_empty2_id;
```

Comment:
Is that name choice intentional or part of the renaming effort? It looks to me as though it will cause build failures.

Reply:
It's a mistake. When I was first adding the new `tx_empty` I called it `tx_empty2` just to keep it distinct from the old `tx_empty`.